docker/cron/dump-crontab

```diff
@@ -1,10 +1,11 @@
+MAILTO=""
```

Comment:
Could we make this `listenbrainz-observability@metabrainz.org`?

Reply:
We could. But the data it would give us would be nearly useless since all the output is re-directed to the log file. Doing so would be an effective way to monitor that cron is running and not much else.

Reply:
BTW, I ran into something similar in AB cron last week, where I got this warning after running a cron job which failed:

> Feb 21 16:52:03 2288dacacfda CRON[64]: (CRON) info (No MTA installed, discarding output)

I'm not sure if there's an MTA installed in the LB base image at the moment, so I think that redirecting output and disabling mail is the right thing to do right at this moment